business_requirement_gap_analysis_task/models/business.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class BusinessRequirementGapAnalysis(models.Model):
+    _inherit = 'business.requirement'
+
+    gap_analysis_task_id = fields.Many2one(
+        comodel_name='project.task',
+        inverse_name='business_requirement_id',
+        string='Gap Analysis Task',
+        help='Use this task to input all the timesheets for the preparation of'
+             ' this BR (meetings, solution design, estimation, etc.)'
+    )
+
+    @api.multi
+    @api.onchange('project_id')
+    def master_project_change(self):
+        for rec in self:
+            if rec.gap_analysis_task_id:
```

Comment:
You can improve these lines with this one:
`if self.mapped('rec.gap_analysis_task_id'):`

Reply:
better:

`for rec in self.mapped('rec.gap_analysis_task_id'):`